app/lib/utils/queryMapper.js

```diff
@@ -134,6 +148,8 @@ function mapServiceChoice(query) {
 module.exports = {
   getLocationHeading,
   getResultsInfo,
+  getResultsInternalLink,
```

Comment:
This function is only used outside of this module for testing. Rather than exporting it on the `queryMapper` module, it would be cleaner to move it to a utility type module so it could still be tested without polluting this module's interface.